tests/ftests/001-cgget-basic_cgget_v1.py

```diff
@@ -23,9 +23,10 @@ def prereqs(config):
     result = consts.TEST_PASSED
     cause = None
 
-    if CgroupVersion.get_version('cpu') != CgroupVersion.CGROUP_V1:
+    if Cgroup.get_cgroup_mode(config) != (Mode.CGROUP_MODE_LEGACY or
+                                          Mode.CGROUP_MODE_HYBRID):
```

Comment:
I think this is working somewhat by luck.

Doing this `Mode.CGROUP_MODE_LEGACY or Mode.CGROUP_MODE_HYBRID` will always result in an integer `1`.  The `1` is then compared against the result from `get_cgroup_mode()`.  Since LEGACY is `1`, it often succeeds.

This can be verified by comparing the number of hybrid tests skipped in this pull request vs. the number of hybrid tests skipped in the main branch.

This pull request:
https://github.com/libcgroup/libcgroup/actions/runs/4707255489/jobs/8349019318?pr=332

Main branch:
https://github.com/libcgroup/libcgroup/actions/runs/4724261724/jobs/8381266627